app/models/post.rb

```diff
@@ -0,0 +1,11 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  validates :content, presence: true, length: { minimum: 10, maximum: 300 }
+  
+  default_scope { order(created_at: :desc) }
+
+  def creator
```

Comment:
This is a good workaround to what could have been achieved in one line right in the `belongs_to` association.

[`OPTIONAL`]